### PR TITLE
Fix filter for thermal result checking for lastTick not for currentTick

### LINKED ISSUE #1008: Fix filter for thermal result checking for lastTick not for currentTick
Looks to me that this rather should be lastTick. Since we yet haven't any ThermalGridIT this comes not directly clear.

' .filter { case (_, state) => state.tick == lastTick}'

_Originally posted by @danielfeismann in https://github.com/ie3-institute/simona/pull/925#discussion_r1820316530_

### PR DESCRIPTION
**Pull request by danielfeismann**
resolves #1008

### CODE REVIEW

File: src/main/scala/edu/ie3/simona/agent/ValueStore.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import scala.collection.SortedMap
   */
 final case class ValueStore[+D](
     maxTickSpan: Long,
-    private val store: SortedMap[Long, D] = SortedMap.empty[Long, D],
+    store: SortedMap[Long, D] = SortedMap.empty[Long, D],
```
Comment (sebastian-peter) [inline src/main/scala/edu/ie3/simona/agent/ValueStore.scala:25]: I think we don't need this anymore?

**Review by sebastian-peter — COMMENTED** [review]
Looks good, if this works for you I'd be ready to merge

File: src/main/scala/edu/ie3/simona/agent/ValueStore.scala (hunk 1 above)
Comment (danielfeismann) [reply src/main/scala/edu/ie3/simona/agent/ValueStore.scala:25]: True, thanks!

**Review by sebastian-peter — APPROVED** [review]
Thanks for this concise and effective PR!

**Pull request merged**